app/src/main/java/com/example/myflix/repository/MovieRepository.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.myflix.repository
+
+import android.util.Log
+import com.example.myflix.api.MovieApi
+import com.example.myflix.api.RetrofitClient
+import com.example.myflix.dto.MovieDetailsDto
+import com.example.myflix.dto.MovieDto
+
+
+object MovieRepository {
```

Comment:
Make class instead of object. You can create an instance of repository using constructor